x/debug.go

```diff
@@ -27,8 +27,24 @@ import (
 	"github.com/dgraph-io/badger/v4"
 	bpb "github.com/dgraph-io/badger/v4/pb"
 	"github.com/dgraph-io/dgraph/protos/pb"
+	"github.com/golang/glog"
```

Comment:
please fix the order of import

Reply:
It's it already correct? Golang should come after dgraph?

Reply:
we separate dgraph imports from non-dgraph imports. Check other files.